location_server.py

```diff
@@ -98,7 +102,7 @@ async def register(websocket, player_name=None, group_key=None):
 
 async def unregister(websocket):
     player_name, group_key = PLAYERS.pop(websocket)
-    if player_name:
+    if player_name or player_name == "":
```

Comment:
Ah heh, empty string is False I guess 😅
This is good defensively, but we should also probably do something about preventing the name from ever being empty on the client side anyway. Maybe if the name is empty there, we go back to the logfile name.